src/pf-tooltip/index.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.13.0/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/pf-utils.js"></script>
+    <script src="/dist/js/pf-tooltip.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+        <h1>Tooltips</h1>
+    </div>
+    <hr>
+
+    <pf-tooltip placement="left" targetSelector="#btn-left">
+        Tooltip text on Left
+    </pf-tooltip>
+    <button id="btn-left" type="button" class="btn btn-primary">Left</button>
+
+    <pf-tooltip placement="top" targetSelector="#btn-top">
+        <span>Tooltip on top</span>
+    </pf-tooltip>
+    <button id="btn-top" type="button" class="btn btn-secondary">Top</button>
+
+    <pf-tooltip placement="bottom" targetSelector="#btn-bottom">
+        <span>Tooltip on bottom</span>
+    </pf-tooltip>
+    <button id="btn-bottom" type="button" class="btn btn-warning">Bottom</button>
+
+    <pf-tooltip placement="right" targetSelector="#btn-right">
+        <span>Tooltip on right</span>
+    </pf-tooltip>
+    <button id="btn-right" type="button" class="btn btn-danger">Right</button>
+
+    <br/>
+    <br/>
+    <script>
+        setTimeout(function(){
+          var pfTooltipRight = document.querySelector('[targetSelector="#btn-right"]');
+          pfTooltipRight.innerHTML = "Right content changed!";
+
+          var event = new CustomEvent('handleContentChanged',{});
+          pfTooltipRight.dispatchEvent(event);
```

Comment:
Per our earlier chat, the innerHTML was only in place here for example sake. A framework would render this content via a binding (like I demonstrated with the React example). I suppose given the option of exposing a method such as "setInnerHtml()" to the shim or using Mutation Observers, I'd choose the former for the perf gain. However this makes things a bit trickier i.m.o. because you'd have to stringify the template after bindings get applied (it works though). The method I've proposed with events would just expose this to the WC within its DOM after the framework finishes rendering inside of its host element. There is no DOM observation in this approach.